sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +211,38 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+            return CreateParameterBindingData(blobAttribute.Connection, blobPath.BlobName, blobPath.ContainerName);
+        }
+
+        private ParameterBindingData ConvertToParameterBindingData(BlobBaseClient input, BlobTriggerAttribute blobTriggerAttribute)
+        {
+            return CreateParameterBindingData(blobTriggerAttribute.Connection, input.Name, input.BlobContainerName);
+        }
+
+        private ParameterBindingData ConvertBlobInputToParameterBindingData(BlobBaseClient input)
+        {
+            return CreateParameterBindingData(null, input.Name, input.BlobContainerName);
+        }
+
+        private ParameterBindingData CreateParameterBindingData(string connection, string blobName, string containerName)
+        {
+            var connectionSection = _blobServiceClientProvider.GetWebJobsConnectionStringSection(connection);
```

Comment:
looks like in your refactoring you lost the name resolver logic. That still needs to be done externally to the call to GetWebJobsConnectionStringSection since that method doesn't do name resolver as you can see. Likely there's a test hole here?

Reply:
Yeah my bad. Updated the code here. It did cause test failures for all ParameterBindingData array - https://dev.azure.com/azure-sdk/public/_build/results?buildId=2033421&view=ms.vss-test-web.build-test-results-tab